### ISSUE TITLE
Working fine locally but not committing when on public server

### ISSUE THREAD

**Issue by ericpedia** [thathoff/kirby-git-content#89]
I have the plugin working brilliantly on my local machine. However, when I make a change on the live site, no commit occurs.

**My config:**

```php
    'thathoff' => [
        'git-content' => [
            'commit' => true,
            'push'=> true,
        ],
    ],
```

**My steps**

1. Edit a page in the panel and click Save.
2. Run `git status` and/or `git log` via ssh from content folder.
    **Expected:** Change has been committed, working directory is clean.
    **Actual:** Changes are visible but have not been staged or uncommitted.

**What I know about the server setup**

- git 2.17.1 is installed
- `composer install` has been run, and all plugin files and vendor packages are there
- git status accurately reflects site changes
- content repo is on `master` branch
- git commands work fine if done directly via command line
- remote origin is Github. SSH keys have been properly set up so that I am able to push to and pull from the server to Github without issues
- Error logs don't show anything, and no error messages appear when saving

**Other notes**

- Local environment works fine, uses PHP 8.1.3
- My content folder is in the normal place
- I have my content repo set up as a submodule (all git commands work as expected inside of it and point to the right remote)

**Kirby Status**

**Debugging:** on
**PHP:** 7.4.25
**Server:** Apache

Any thoughts?

**Comment by ericpedia** [comment]
I have also confirmed that PHP has access to git. I created `test.php` inside the content folder and tested `shell_exec('git add -A');`. It successfully ran the git command and staged all unstaged files.

**Comment by thathoff** [comment]
The plugin acts as the webserver’s user. So this user needs to be able to make git commit, git push and pull actions. 

Have you tried to enable the  `displayErrors` option? In this case you might get some information about errors happening when committing.

**Comment by ericpedia** [comment]
Yeah, my config has `displayErrors` and `debug` both set to true, but no errors appear in the Kirby interface, browser console, or server error log, as far as I can tell.

I also confirmed that the webserver user on my Kirby site can successfully use git, by adding various operations like `exec('git add -A')` to my Kirby templates, which worked (confirmed by checking content repo status directly via SSH afterwards). Do you have any ideas of how to troubleshoot it?

**Comment by thathoff** [comment]
With `displayErrors` is an option for this plugin (not the php.ini option). 

You said your config looks like this:

```
'thathoff' => [
    'git-content' => [
        'commit' => true,
        'push'=> true,
    ],
],
```

try to change it to

```
'thathoff' => [
    'git-content' => [
        'commit' => true,
        'push'=> true,
        'displayErrors' => true,
    ],
],
```

**Comment by thathoff** [comment]
Closing this issue because it has not been updated for a longer time. Considering it as resolved. Otherwise feel free to re-open.
